### ISSUE TITLE
When all code reviews are visible, "visible code reviews" dropdown should be hidden from modify course page

### ISSUE THREAD

**Issue by mjaakko** [UniversityOfHelsinkiCS/labtool#1012]
Currently the dropdown is visible, but when it is clicked, nothing happens. This behavior is confusing.

Quick fix would be to hide dropdown menu when all code reviews are visible.

**Comment by hisahi** [comment]
A proper fix is to improve the entire UI

**Comment by yumoL** [comment]
> A proper fix is to improve the entire UI

What else of the UI should be improved?

**Comment by hisahi** [comment]
> 
> 
> > A proper fix is to improve the entire UI
> 
> What else of the UI should be improved?

For example, it could be a list of checkboxes instead of what it is now. The list could also be inside a frame so that you'd have to scroll if there are too many code reviews, so that even having 100 code reviews won't mean you would have to scroll through the entire page

**Comment by hisahi** [comment]
Done in #1013
